### ISSUE TITLE
GPU used for pre-training

### ISSUE THREAD

**Issue by remcofl** [qizekun/ReCon#20]
Dear authors,

In the paper, you mention the GPU hours needed for pre-training on the different model variants (table 18). These were performed using a single GPU instance. Do you recall which GPU was used for this study? And were these times achieved with mixed precision training?

**Comment by qizekun** [comment]
We train our models using a single NVIDIA 3090 or 4090 GPU.
**Mixed precision training should be strictly avoided**, as it leads to non-converging loss and does not work in the context of MAE.

**Comment by remcofl** [comment]
Thank you for the fast response!

Regarding mixed precision training, I wonder if it helps to keep the final projection and chamfer dist calculations in FP32. I will look into it.
